packages/python-packages/apiview-copilot/tsp/main.tsp

```diff
@@ -60,11 +73,23 @@ namespace ApiViewCopilot {
     /** The line number of the comment */
     lineNo: int32;
 
-    /** The author of the comment */
-    author: string;
+    /** The datetime the comment was created */
+    createdOn: offsetDateTime;
+
+    /** The count of upvotes */
+    upvotes?: int32;
+
+    /** The count of downvotes */
+    downvotes?: int32;
+
+    /** The creator of the comment */
+    createdBy: string;
```

Comment:
@tjprescott , this is currently `author` just double checking if changing this name will break anything on the `review` side, I don't know if you are already using this code :D